security/seccomp-profiles.adoc

```diff
@@ -0,0 +1,77 @@
+[id="seccomp-profiles"]
+= Configuring seccomp profiles
+include::modules/common-attributes.adoc[]
+:context: seccomp-profiles
+
+toc::[]
+
+A container or an OpenShift pod runs a single application that performs one or more well-defined tasks. The application usually requires only a small subset of the underlying operating system kernel APIs.
+Seccomp (secure computing mode) Linux kernel feature can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating profile that is named as seccomp profile.
+
+?? Are below statements important or just a note?
+[IMPORTANT]
+====
+CRI-O runs unconfined by default, without any seccomp profile applied.
+====
+
+[IMPORTANT]
+====
+Seccomp profile cannot be applied to privileged containers.
+====
+
+== Configuring default seccomp profile
+Openshift ships with a default seccomp profile. On RHEL CoreOS, the default seccomp profile is stored in `/etc/crio/seccomp.json`. You can enable the default seccomp profile by overriding the empty seccomp value with `runtime/default`.
+
+
+.Sample default seccomp profile
+[%collapsible]
+====
+[source,json]
+----
+include::https://raw.githubusercontent.com/containers/common/main/pkg/seccomp/seccomp.json[]
+----
+//Add link to the appendix above
+//Alternate link found https://github.com/moby/moby/blob/master/profiles/seccomp/default.json
+====
+== Configuring a custom seccomp profile
+You can configure a custom seccomp profile, which allows you to update the filters based on the application requirements. This allows cluster administrators to have greater control over the security of workloads running in OpenShift Container Platform.
+[NOTE]
+====
+You can refer to default seccomp profile stored in `/etc/crio/seccomp.json` to create a custom seccomp profile.
+====
+
+.Prerequisite
+* You have created a custom SCC. For more information, see "Additional resources".
```

Comment:
Can we link to an explanation about what an SCC is?

Reply:
There's a link to SCC topic in Additional resources section that is being referred to.

Reply:
Ah good did not see that one. Then let's just reference the abbreviation:
```suggestion
* You have created a custom Security Context Constraints (SCC). For more information, see "Additional resources".
```